panel/param.py

```diff
@@ -761,14 +761,11 @@ def select(self, selector=None):
         return super().select(selector) + self.layout.select(selector)
 
 
-class ParamMethod(ReplacementPane):
+class ParamRef(ReplacementPane):
```

Comment:
Should we use `ParamRef`, or is it private or abstract?

Reply:
Neither private nor abstract.